src/main/resources/metrics-config/labels.yaml

```diff
@@ -0,0 +1,204 @@
+mappers:
```

Comment:
Should it be prometheus.yaml or properties.yaml with prometheus.mappers structure?